### DISCUSSION TITLE
404: NOT_FOUND Code: NOT_FOUND

### DISCUSSION THREAD

**Discussion by caronqq** [vercel/community#5075] [Help]
### Page to Investigate

https://stable-video-diffusion.live/

### Steps to Reproduce

404: NOT_FOUND
Code: NOT_FOUND
ID: sfo1::nd4fj-1701776538982-9a0a4ec7e775

[GET]

/favicon.ico

Time
December 05 19:42:19.00 GMT+08:00
Request Path
/favicon.ico
Status Code
404
Host
stable-video-diffusion-live-n2akcvh9l-carons-projects.vercel.app
Request ID
gwtx6-1701776539944-a2c76ed567d7

**Comment by amyegan** [comment]
Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) we can use to help you debug this [404: NOT_FOUND](https://vercel.com/docs/errors/platform-error-codes#404:-not_found) error?
